### PR TITLE
CEDS-2346 Convert page layout to GDS Design System

### PR DESCRIPTION
**Pull request by PauloPortugal**
**Depends on** https://github.com/hmrc/cds-exports-acceptance/pull/418
_(Smoke tests are passing locally)_

### CODE REVIEW

File: app/views/declaration/supervising_customs_office.scala.html [hunk 1]
```diff
@@ -14,50 +14,78 @@
...
+    @components.details(messages("site.details.summary_text_these")) {
+        @Html(messages("declaration.type.supervisingCustomsOfficeSummaryText", components.details_content_link()))
+    }
+}
 
-            @components.back_link(Navigator.backLink(SupervisingCustomsOffice, mode)(request))
+@autocomplete = {
+    @components.fields.field_accessible_autocomplete(
+        field = form("supervisingCustomsOffice"),
+        label = "",
```
Comment (PatrykRudnicki) [inline app/views/declaration/supervising_customs_office.scala.html:60]: I'm thinking if the empty label cause any accessibility issues.

File: app/views/declaration/supervising_customs_office.scala.html [hunk 2]
```diff
@@ -14,50 +14,78 @@
...
+        @errorSummary(form.errors)
 
-            @components.fields.field_autocomplete(
-                field = form("supervisingCustomsOffice"),
-                label = messages(""),
-                labelClass = Some("form-label-bold"),
-                hintText = None,
-                emptySelectValue = messages("supplementary.packageInformation.typesOfPackages.empty"),
-                items = AutoCompleteItem.fromSupervisingCustomsOffice(CustomsOffices.all))
+        @sectionHeader(messages("declaration.warehouse.supervisingCustomsOffice.sectionHeader"))
```
Comment (PatrykRudnicki) [inline app/views/declaration/supervising_customs_office.scala.html:75]: I think that the secitonHeader can be moved to the autocomplete as label with custom class, but not sure about it.

File: app/views/declaration/supervising_customs_office.scala.html (hunk 1 above)
Comment (PauloPortugal) [reply app/views/declaration/supervising_customs_office.scala.html:60]: Yes I agree, but the prototype does not have one. I have asked this question to Chris, but he said that the screen reader does not find a problem

File: app/views/declaration/supervising_customs_office.scala.html (hunk 2 above)
Comment (PauloPortugal) [reply app/views/declaration/supervising_customs_office.scala.html:75]: Hum... I am not sure about that, as this is the way we tend to do things. Admittedly there are two labels missing as part of the autocomplete component: one for the select tag and another for the input text tag.

**Pull request merged**